api/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'war'
+}
+
+group 'com.epam.esm'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':core')
+
+    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.0'
+    testCompile group: 'org.springframework', name: 'spring-test', version: '5.3.2'
+    compile group: 'org.springframework', name: 'spring-core', version: '5.3.2'
+    compile group: 'org.springframework', name: 'spring-beans', version: '5.3.2'
+    compile group: 'org.springframework', name: 'spring-context', version: '5.3.2'
+    compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.16'
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.16'
+    compile group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
+    compile group: 'org.springframework', name: 'spring-web', version: '5.3.2'
+    compile group: 'org.springframework', name: 'spring-webmvc', version: '5.3.2'
+    providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version: '4.0.1'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.3'
+    compile group: 'org.springframework', name: 'spring-jdbc', version: '5.3.2'
+    compile group: 'org.postgresql', name: 'postgresql', version: '42.2.18'
+    compile group: 'joda-time', name: 'joda-time', version: '2.9.9'
+    testCompile group: 'com.h2database', name: 'h2', version: '1.4.200'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '3.6.28'
+    testCompile group: 'org.mockito', name: 'mockito-junit-jupiter', version: '3.6.0'
+
+    compile group: 'com.zaxxer', name: 'HikariCP', version: '3.3.1'
+    compile group: 'io.swagger.core.v3', name: 'swagger-annotations', version: '2.1.2'
```

Comment:
What the difference between implementation and compile configurations?

Reply:
the implementation dependency configuration provides the same functionality as compile, but compile configuration is deprecated, fixed it